CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+Thank you for taking the time to contribute to JustiFi Web Components! The following is a set of guidelines for contributing to this project.
```

Comment:
Instead of "Thank you for taking the time to contribute to JustiFi Web Components!", maybe "Thank you for taking the time to contribute to the JustiFi Web Component Library!"? Small change but I feel like it reads slightly better maybe

Reply:
Good call on these edits - thanks!